diagnostic.js

```diff
@@ -32,26 +41,48 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update({}, {
+  $set: {
+    "members": []
+  }
+}, false, true);
+db.houses.update({
+  name: "Winterfell"
+}, {
+  $push: [{
+    members: "Ned Stark"
+  }, {
+    members: "Arya Stark"
+  }, {
+    members: "Sansa Stark"
+  }]
+});
```

Comment:
1) line 44: 1st argument is blank, so it won't match anything
2) line 48: what are `false` and `true`?
3) what is Winterfell? Does it exist in your database?
4) for $push, you are pushing an array. Does $push accept an array as an argument?

check out this solution:

```js
db.houses.update(
  {name: "House Stark"},
  {$set: { members: { [ "Ned Stark", "Arya Stark", "Sansa Stark"] }}}
);
```